Platform/AlderlakeBoardPkg/BoardConfigAdlN.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, *args, **kwargs):
         self.FSP_IMAGE_ID         = 'ADLI-FSP'
         self._EXTRA_INC_PATH      = ['Silicon/AlderlakePkg/Adln/Include']
         self._FSP_PATH_NAME       = 'Silicon/AlderlakePkg/Adln/FspBin'
+        self.FSP_INF_FILE         = 'Silicon/AlderlakePkg/FspBin/FspBin.inf'
```

Comment:
This is the default path. it is optional to add it.

Reply:
ok, removed.